src/Tree.hpp

```diff
@@ -54,10 +56,18 @@ namespace RRT
             return _state;
         }
 
+        /**
+         * The distance from this node to its parent.
+         */
+        float distance() const {
```

Comment:
I don't think this method or the `_dist` instance variable are needed anymore not that the `intermediateState()` method has changed.